README.md

```diff
@@ -1 +1,240 @@
-# budget-app
\ No newline at end of file
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+
+![mobile](https://user-images.githubusercontent.com/105460631/220864252-c1381fdf-35cd-4760-b6dd-8262b36e04e6.png)
+![mobile (1)](https://user-images.githubusercontent.com/105460631/220864282-a4993499-95fb-49a8-ab37-e3b5eca242f2.png)
+![mobile (2)](https://user-images.githubusercontent.com/105460631/220864316-80da1d04-f9db-4fe7-a079-8367562bdf3b.png)
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+E-neza App is a frinchised online web application that allows the user to manage his/her budget: they have a list of transactions associated with a category, so that the user can see how much money they spend and on a particular category.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Rails Default Server</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Create, show or delete a users
+- Modify styles towaards top level
+- Display a transaction list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://www.loom.com/share/a0bca297074c48509902e8e8a65cb00b)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
```

Comment:
- [ ] Kindly remove this section description to make your readme file descriptive and clean :+1: These are just instructions for the owner of the repository so they should not be in the readme file :thinking: